src/docs/tutorial-branding/working-with-translations/index.md

```diff
@@ -3,7 +3,9 @@ title: Working with translations
 slug: working-with-translations
 updated: 2020-02-28
 category: tutorial-branding
-ingress: Write text content for your marketplace
+ingress:
+  A tutorial for changing the UI copy-texts and implementing new
```

Comment:
Would it make sense to have all the tutorial ingresses in the same imperative format, e.g. 'Change the UI copy texts...'

Reply:
Good point. This already seems to be applied in most tutorials so makes sense to use the same style here. Also, in the future if we're updating a tutorial article, the ingress can always be updated to match the same format if it already does not.